qa/tasks/cephfs/mount.py

```diff
@@ -100,10 +100,13 @@ def __init__(self, ctx, test_dir, client_id, client_remote,
     # last failed test cases.
     @staticmethod
     def cleanup_stale_netnses_and_bridge(remote):
+        log.info("About to run 'ip netns list'...")
         p = remote.run(args=['ip', 'netns', 'list'],
-                       stdout=StringIO(), timeout=(5*60))
+                                stderr=StringIO(), stdout=StringIO(),
+                                timeout=(5*60))
+        if p.stderr.getvalue().strip():
+            log.warning(f"ip netns list stderr: {p.stderr.getvalue()}")
```

Comment:
And nothing showed up in https://jenkins.ceph.com/job/ceph-api/102980/consoleFull#212386344640526d21-3511-427d-909c-dd086c0d1034 o_O

Reply:
That's a pretty old defect report. I would be surprised if we are running into exactly that issue.